content/cuda.md

```diff
@@ -16,7 +16,122 @@ tableOfContents: true
 
 ## Pop!\_OS 22.04 LTS
 
-It is recommended to use [Tensorman](/articles/tensorman) as newer versions of CUDA are no longer packaged on their own.
+If you are looking to install other versions of the CUDA libraries other than the version included with the Nvidia driver, we suggest using the `nvidia-container-toolkit` to allow alternate versions of the CUDA libraries to be installed. This should give you native speed that was tested to work by Nvidia as they package the libraries in a docker image. You can see the different iamages that are published here: <https://hub.docker.com/r/nvidia/cuda/>
```

Comment:
`other versions of the CUDA libraries other than` is worded redundantly.

`Nvidia driver` should be `NVIDIA driver`.

`This should give you native speed that was tested to work` is both unsurely worded ("should") and grammatically incorrect. `docker image` should also be capitalized (`Docker image`) since Docker is a brand.

`iamages` is misspelled.

Reply:
Addressed in [d5f346e](https://github.com/system76/docs/pull/1119/commits/d5f346ea9bc43b54a43fc43378664abaff2a142c).

The entire sentence `This should give you native speed that was tested to work by Nvidia as they package the libraries in a docker image.` seemed unnecessary after moving the key point (that it's packaged directly by NVIDIA) to another sentence. I included Docker earlier in the paragraph since the following guide is really talking about how to use Docker more than anything else.